CMakeLists.txt

```diff
@@ -154,6 +154,10 @@ string(REGEX MATCH "Clang" CMAKE_COMPILER_IS_CLANG "${CMAKE_C_COMPILER_ID}")
 
 include(CheckCCompilerFlag)
 
+if (NOT CMAKE_VERSION VERSION_LESS 3.1.0)
```

Comment:
Why do we need that?

Reply:
`CMAKE_C_STANDARD` is available from cmake 3.1
ref: https://cmake.org/cmake/help/v3.1/release/3.1.0.html